css/css-flexbox-1/negative-margins-001.html

```diff
@@ -4,11 +4,9 @@
 <link rel="author" title="Google Inc." href="https://www.google.com/">
 <link rel="help" href="http://www.w3.org/TR/css-flexbox-1/#item-margins">
 <link rel="match" href="reference/negative-margins-001-ref.html">
-<meta name="assert" content="Tests that for intrinsic size computations,
```

Comment:
Why are you deleting this?

Reply:
Not sure what happened here? Wasn't in the PR this was split out from...